services/ui/ws/external_window_display_root_window.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SERVICES_UI_WS_EXTERNAL_WINDOW_DISPLAY_ROOT_WINDOW_H_
+#define SERVICES_UI_WS_EXTERNAL_WINDOW_DISPLAY_ROOT_WINDOW_H_
+
+#include "services/ui/ws/server_window.h"
+
+#include "components/viz/common/surfaces/frame_sink_id.h"
+#include "ui/gfx/geometry/rect.h"
+
+namespace ui {
+namespace ws {
+
+class WindowServer;
+
```

Comment:
Add comment what this class does and why it's needed.

Reply:
Done